LondonBritishLibrary/orient/BLorient618.xml

```diff
@@ -0,0 +1,204 @@
+<title type="complete" xml:lang="en">Homily of John Chrysostom on the Annunciation of the blessed Virgin</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.5">
+<locus from="45v"/>
+<title type="complete" xml:lang="en">Another homily of Chrysostom on the same subject</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.6">
+<locus from="49v"/>
+<title type="complete" xml:lang="en">Homily of Jacob of Sarug on the same subject</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+</msItem>
+
+<msItem xml:id="ms_i2">
+<locus from="54r"/>
+<title type="complete" ref="LIT1297Dersan"/>
+<textLang mainLang="gez"></textLang>
+
+<msItem xml:id="ms_i2.1">
+<locus from="54r"/>
+<title type="complete" ref="LIT1297Dersan#HomilyChrysostom"/>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2.2">
+<locus from="71v"/>
+<title type="complete" ref="LIT1297Dersan#Miracles"/>
+<note>The history of Ṭobit on <locus target="#76v">f. 76v</locus>.</note>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+</msItem>
+
+</msContents>
+
+<physDesc>
+  <objectDesc form="Codex">
+      <supportDesc>
+          <support>
+              <material key="parchment"/>
+          </support>
+          <extent>
+              <measure unit="leaf">98</measure>
+              <measure unit="leaf" type="blank">2</measure>
+<dimensions type="outer" unit="in">
+<height>11.75</height>
+<width>10.125</width>
+</dimensions>
+
+            </extent></supportDesc>
+
+            <layoutDesc>
+                <layout columns="2" writtenLines="19">
+                </layout>
+            </layoutDesc>
+
+</objectDesc>
+
+<handDesc>
+    <handNote script="Ethiopic" xml:id="h1">
+        <desc>Written in a good hand, of the earlier part of the 18th Century.</desc>
+        <date notBefore="1700" notAfter="1750" resp="PRS10171Wright"/>
+    </handNote>
+</handDesc>
+
+<additions>
+<list>
+  <item xml:id="a1">
+  <locus target="#1r"></locus>
+    <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+            <placeName ref="INS0101MadhaneAlam"/>.</desc>
+        <q xml:lang="gez">ድርሳነ፡ ገብርኤል፡ ዘ<placeName ref="INS0101MadhaneAlam"> ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName></q>
+</item>
+
+<item xml:id="e1">
+<locus target="#53v"/>
+<desc>The name of the scribe is <persName role="scribe" ref="PRS13527Fiqtor"><foreign xml:lang="gez">ፊቅጦር፡</foreign></persName>.</desc>
+</item>
+
+<item xml:id="e2">
+<locus target="98"/>
+<desc>Writing on <locus target="#53v">f. 98</locus> erased.</desc>
+</item>
+
+<item xml:id="e3">
+<locus target="#1r #97r"/>
+<desc>Name of the first owner erased, and that of <persName role="owner" ref="PRS13518KeflaGiyorgis"><foreign xml:lang="gez">ክፍለ፡ ጊዮርጊስ፡</foreign></persName> substituted.</desc>
+</item>
+
+<item xml:id="e4">
+<desc>Later owners: <persName role="owner" ref="PRS13519MalkeaIyasus"><foreign xml:lang="gez">መልክአ፡ ኢየሱስ፡</foreign></persName>,
```

Comment:
In another commit PRS13519MalkeaIyasus is encoded as an owner of BLorient 611. This commit is already merged. So please, if there is some confusion creat a new branch for corrections. The same is at least for the two other.

Reply:
I assumed that four names which appear both in BLorient611 and BLorient 618 are the same people, it's too much of a coincidence imo.

Reply:
Thank you! Would you mind adding the relation to this manuscript in their files as well?